packages/core/contracts/zodiac/OptimisticGovernor.sol

```diff
@@ -95,6 +95,7 @@ contract OptimisticGovernor is Module, Lockable {
             bytes32 _identifier,
             uint64 _liveness
         ) = abi.decode(initializeParams, (address, address, address, uint256, string, bytes32, uint64));
+        require(_finder != address(0), "finder address can not be empty");
         finder = FinderInterface(_finder);
         require(_getCollateralWhitelist().isOnWhitelist(address(_collateral)), "bond token not supported");
```

Comment:
can you group the `requires` all together at the top of the function rather than doing require then assignment.

Reply:
@chrismaree I think this gets removed in a different fix, so we should avoid moving it to avoid a merge conflict.